Extension/package.nls.json

```diff
@@ -951,6 +951,33 @@
+    "c_cpp.debuggers.cpplldb.detachOnError.description": "Detach the debugger when an error occurs. ",
+    "c_cpp.debuggers.cpplldb.debuggerRoot.description": "The directory to where relative object files can be located.",
+    "c_cpp.debuggers.cpplldb.targetTriple.description": "Overrides the triplet of the target program",
+    "c_cpp.debuggers.cpplldb.platformName.description": "Overrides the platform name of the target program",
+    "c_cpp.debuggers.cpplldb.initCommands.description": "LLDB commands executed upon debugger startup prior to creating the LLDB target.",
+    "c_cpp.debuggers.cpplldb.preRunCommands.description": "LLDB commands executed just before launching/attaching, after the LLDB target has been created.",
+    "c_cpp.debuggers.cpplldb.postRunCommands.description": "LLDB commands executed after launching when it's in a stopped state prior to any automatic continuation.",
+    "c_cpp.debuggers.cpplldb.launchCommands.description": "Custom LLDB commands that are executed instead of launching a process.",
+    "c_cpp.debuggers.cpplldb.stopCommands.description": "LLDB commands executed each time the program stops.",
+    "c_cpp.debuggers.cpplldb.exitCommands.description": "LLDB ommands executed when the program exits.",
+    "c_cpp.debuggers.cpplldb.terminateCommands.description": "LLDB commands executed when the debugging session ends.",
+    "c_cpp.debuggers.cpplldb.enableAutoVariableSummaries.description": "Enable auto generated summaries for variables when no summaries exist for a given type. This feature can cause performance delays in large projects when viewing variables.",
+    "c_cpp.debuggers.cpplldb.displayExtendedBacktrace.description": "Enable language specific extended backtraces.",
+    "c_cpp.debuggers.cpplldb.enableSyntheticChildDebugging.description": "If a variable is displayed using a synthetic children, also display the actual contents of the variable at the end under an entry. This is useful when creating sythetic child plug-ins as it lets you see the actual contents of the variable.",
+    "c_cpp.debuggers.cpplldb.commandEscapePrefix.description": "The escape prefix character to use for executing regular LLDB commands in the Debug Console, instead of printing variables.",
+    "c_cpp.debuggers.cpplldb.customFrameFormat.description": "When specified, stack frames will have descriptions generated based on the provided format. See https://lldb.llvm.org/use/formatting.html",
+    "c_cpp.debuggers.cpplldb.customThreadFormat.description": "When specified, threads will have descriptions generated based on the provided format. See https://lldb.llvm.org/use/formatting.html",
+    "c_cpp.debuggers.cpplldb.attachCommands.description": "Custom LLDB commands that are executed instead of attaching to a process ID or to a process by name.",
+    "c_cpp.debuggers.cpplldb.waitFor.description": "Wait for the process to launch by looking for a process with a basename that matches `program`.",
```

Comment:
What is `program`?

Reply:
The `program` field in the debug configuration